### PR TITLE
Disable YaST self-update in the quarterly media updates (bsc#1150012)

### PR DESCRIPTION
**Pull request by lslezak**
- Same as in https://github.com/yast/skelcd-control-leanos/pull/39, just for SP2

### CODE REVIEW

File: Rakefile [hunk 1]
```diff
@@ -1,8 +1,11 @@
 require "yast/rake"
 
-Yast::Tasks.submit_to :sle15sp2
```
Comment (imobachgs) [inline Rakefile:3]: Should we consider adding a `sle15sp2-qr` target or something similar? Or is it maybe too much? In any case, it is not something that we need to decide now.

**Review by imobachgs — APPROVED** [review]

File: Rakefile (hunk 1 above)
Comment (imobachgs) [reply Rakefile:3]: If it only affects this package, then I guess it is fine to do not add more targets.

Comment (lslezak) [reply Rakefile:3]: We need only this single package for the QR media, all other changes should normally go to SP2 MU. Adding `sle15sp2-qr` would need an update for `yast-rake`, you need to ensure it's updated in the Jenkins workers, etc... I think it's easier to just modify the `Rakefile` here.

**Comment by lslezak** [comment]
Submitted in https://build.suse.de/request/show/226570

**Pull request merged**